src/vdbeapi.c

```diff
@@ -1108,53 +1197,52 @@ static void columnMallocFailure(sqlite3_stmt *pStmt)
 ** The following routines are used to access elements of the current row
 ** in the result set.
 */
-
-void sqlite3_column_blob_(sqlite3_stmt *pStmt, int i) __EXPORT_NAME(sqlite3_column_blob) {
+void sqlite3_column_blob_(sqlite3_stmt *pStmt, int i) __attribute__((export_name("sqlite3_column_blob"))) {
    const char *blob = sqlite3_column_blob(pStmt, i);
    int blob_len = sqlite3_column_bytes(pStmt, i);
 
    // blob is managed by SQLite itself
    set_result_ptr((void *)blob);
    set_result_size(blob_len);
 }
-
-const void *sqlite3_column_blob(sqlite3_stmt *pStmt, int i){
+const void *sqlite3_column_blob(sqlite3_stmt *pStmt, int i) {
   const void *val;
-  val = sqlite3_value_blob( columnMem(pStmt,i) );
+  val = sqlite3_value_blob(columnMem(pStmt, i));
   /* Even though there is no encoding conversion, value_blob() might
-  ** need to call malloc() to expand the result of a zeroblob() 
-  ** expression. 
+  ** need to call malloc() to expand the result of a zeroblob()
+  ** expression.
   */
   columnMallocFailure(pStmt);
   return val;
 }
-int sqlite3_column_bytes(sqlite3_stmt *pStmt, int i){
-  int val = sqlite3_value_bytes( columnMem(pStmt,i) );
+int sqlite3_column_bytes(sqlite3_stmt *pStmt, int i) {
+  int val = sqlite3_value_bytes(columnMem(pStmt, i));
   columnMallocFailure(pStmt);
   return val;
 }
-int sqlite3_column_bytes16(sqlite3_stmt *pStmt, int i){
-  int val = sqlite3_value_bytes16( columnMem(pStmt,i) );
+int sqlite3_column_bytes16(sqlite3_stmt *pStmt, int i) {
+  int val = sqlite3_value_bytes16(columnMem(pStmt, i));
   columnMallocFailure(pStmt);
   return val;
 }
-double sqlite3_column_double(sqlite3_stmt *pStmt, int i){
-  double val = sqlite3_value_double( columnMem(pStmt,i) );
+double sqlite3_column_double(sqlite3_stmt *pStmt, int i) {
+  double val = sqlite3_value_double(columnMem(pStmt, i));
   columnMallocFailure(pStmt);
   return val;
 }
-int sqlite3_column_int(sqlite3_stmt *pStmt, int i){
-  int val = sqlite3_value_int( columnMem(pStmt,i) );
+int sqlite3_column_int(sqlite3_stmt *pStmt, int i) {
+  int val = sqlite3_value_int(columnMem(pStmt, i));
   columnMallocFailure(pStmt);
   return val;
 }
-sqlite_int64 sqlite3_column_int64(sqlite3_stmt *pStmt, int i){
-  sqlite_int64 val = sqlite3_value_int64( columnMem(pStmt,i) );
+sqlite_int64 sqlite3_column_int64(sqlite3_stmt *pStmt, int i) {
+  sqlite_int64 val = sqlite3_value_int64(columnMem(pStmt, i));
   columnMallocFailure(pStmt);
   return val;
 }
 
-void sqlite3_column_text_(sqlite3_stmt *pStmt, int i) __EXPORT_NAME(sqlite3_column_text) {
+void sqlite3_column_text_(sqlite3_stmt *pStmt, int i)
```

Comment:
let's wrap this function with `#ifndef __sqlite_unmodified_upstream`

Reply:
Nice catch.